Capstone/Feed/HomeViewController.m

```diff
@@ -0,0 +1,112 @@
+//
+//  HomeViewController.m
+//  Capstone
+//
+//  Created by Abel Kelbessa on 7/4/22.
+//
+
+#import "HomeViewController.h"
+#import "HomeCell.h"
+#import "UIImageView+AFNetworking.h"
+#import "DetailsViewController.h"
+#import "APIManager.h"
+
+@interface HomeViewController ()<UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate>
+
+@property(strong, nonatomic) NSArray<NSDictionary *> *colleges;
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (strong, nonatomic) NSMutableArray *collegeDictionaryArrays;
+@property (weak, nonatomic) IBOutlet UISearchBar *searchBar;
+@property (strong, nonatomic) NSMutableArray *filteredColleges;
+
+@end
+
+bool isFiltered;
+
+@implementation HomeViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+    self.searchBar.delegate = self;
+    isFiltered = false;
+    self.collegeDictionaryArrays = [[NSMutableArray alloc] init];
+    [self fetchData];
+}
+
+- (void)fetchData {
```

Comment:
Bring back the TODO to move this to the APIManager?

or, significantly better, just move this to the APIManager 🥹